build/build-sdk-images/node/Dockerfile

```diff
@@ -14,7 +14,7 @@
 ARG BASE_IMAGE=agones-build-sdk-base:latest
 FROM $BASE_IMAGE
 
-RUN curl -sL https://deb.nodesource.com/setup_11.x | bash - && \
+RUN curl -sL https://deb.nodesource.com/setup_12.x | bash - && \
```

Comment:
Oh I didn't spot this before but generally best to stick with the LTS version which will always be even and is currently 12. So just for future reference so we don't upgrade to 13